File: python/exercises/exercise 13.py
import os
from dataclasses import dataclass

os.system("cls||clear")

QUANTIDADE_ALUNOS = 2

# Classe padrão
#class Aluno:
    #def __init__(self, nome, idade):
        #self.nome = nome
        #self.idade = idade
        
# Classe com import
@dataclass
class Aluno:
    nome: str
    idade: int
    peso: float
    altura: float
    
alunos = []

for i in range(QUANTIDADE_ALUNOS):
    nome = input("Digite seu nome: ")
    idade = int(input("Digite sua idade: "))
    peso = float(input("Digite seu peso: "))
    altura = float(input("Digite sua altura: "))
    aluno = Aluno(nome = nome, idade = idade, peso = peso, altura = altura)
    alunos.append(aluno)
    
for dados_aluno in alunos:
    print(f"Nome: {dados_aluno.nome}")
    print(f"Idade: {dados_aluno.idade}")
    print(f"Peso: {dados_aluno.peso}")
    print(f"Altura: {dados_aluno.altura}")

File: c/exercises/exercise13.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>
#include <ctype.h>

int main () {
    setlocale(LC_ALL, "");
    
    char sexo;
    float peso, altura;
    
    printf("Digite seu sexo: ");
    scanf("%c",&sexo);
    
    printf("Digite sua altura: ");
    scanf("%f",&altura);
    
    system("cls||clear");
    
    sexo = toupper (sexo); // Converte em maiúsculo

    switch (sexo) {
    	case 'M':
    		peso = (72.7 * altura) - 58;
    		printf("=== Exibindo Resultados ===\n");
            printf("Seu peso ideal é: %.2f \n", peso);
            break;
        case 'F':
        	peso = (62.1 * altura) - 44.7;
        	printf("=== Exibindo Resultados ===\n");
            printf("Seu peso ideal é: %.2f \n", peso);
            break;
        default :
        	printf("Sexo indefinido");
    }
			
     return 0;
	}
